File: src/main/scala/br/com/fabioaraujo/calcula/produto/dao/HoraTrabalhadaRepository.scala
package br.com.fabioaraujo.calcula.produto.dao

import br.com.fabioaraujo.calcula.produto.dto.HoraTrabalhada
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
trait HoraTrabalhadaRepository extends CrudRepository[HoraTrabalhada, java.lang.Long] {

}

File: src/main/java/br/com/fabioaraujo/calcula/produto/dao/HoraTrabalhadaRepository.java
package br.com.fabioaraujo.calcula.produto.dao;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

import br.com.fabioaraujo.calcula.produto.dto.HoraTrabalhada;

@Repository
public interface HoraTrabalhadaRepository extends CrudRepository<HoraTrabalhada, Long> {

}